app/src/main/java/org/glucosio/android/presenter/AddGlucosePresenter.java

```diff
@@ -112,7 +110,7 @@ private boolean createReading(String type, String notes, long oldId, Date finalD
     }
 
     public double convertToMmol(int mgDl) {
```

Comment:
Having one line method in presenter that has almost same name as in converter doesn't have much sense. So I would inline method in presenter.